academy/3-ibc/4-clients.md

```diff
@@ -251,6 +255,8 @@ As shown in the deep dive on [channels](/academy/3-ibc/3-channels.md), a relayer
 
 The code snippet which illustrates how a client [verifies an incoming packet](https://github.com/cosmos/ibc-go/blob/main/modules/light-clients/07-tendermint/client_state.go) is as follows:
 
+<!-- TODO: update for client refactor v7 -->
```

Comment:
Is there an issue to track this?

Reply:
Not yet, still in the process of streamlining how to do continuous improvement on the dev portal ibc content:
- by quarter 
- or have a tracking issue on the ibc-go repo even to go through dev portal content whenever there's an ibc-go release

Reply:
Created  #1278 to track